test.js

```diff
@@ -197,6 +197,17 @@ test('SR-Flip-Flip : No Change', t => {
   clock.switchOff()
 })
 
+test('T-Flip-Flip: 1', t => {
+  const T = wires(1)
+  const qqbar = wires(2)
+  const clock = new Pulse(500, 1)
+  clock.switchOn()
+  const ff = new TFlipFlop(T, qqbar, clock)
+  const ioHandler = new StringIO(ff)
+  t.is(ioHandler.input('1'), '1')
+  clock.switchOff()
+})
+
```

Comment:
As this library is in it's infancy, there might be some problems with how it handles building components from other components. Please do debug whether all the wires have appropriate values when a simulation is being run.

Also, as both qqbar[0] and qqbar[1] have an initial value of `undefined`, maybe that's why your component is not able to toggle it.